### DISCUSSION TITLE
List of available languages

### DISCUSSION THREAD

**Discussion by noraksp** [BenWiseman/sentiment.ai#8] [Q&A] [answered]
Hi Ben,
this package looks really fantastic and useful! I would be interested in using it with Swedish texts, unfortunately, I couldn't find more detailed information which are the 16 languages it can actually handle, is there a list? And to use it with any language other than English, do I use the multi model or is there an option to choose a model for a specific language?
Thanks,
Nora

**Comment by BenWiseman** [comment, marked as answer]
Thanks!

The languages supported by the multi-lingual embedding are Arabic, Chinese-simplified, Chinese-traditional, English, French, German, Italian, Japanese, Korean, Dutch, Polish, Portuguese, Spanish, Thai, Turkish, and Russian. The embedding isn't language specific, so it will be less effective than a language specific model. 

However, if possible, I've found it better to translate to English and then use the large English (default) model rather than have the embedding tacitly translate the content. Microsoft Azure and Google have free tiers for their translation APIs which, depending on your data and coding patience, may be helpful for your use case.

**Reply by noraksp** [reply]
Great, thanks for the answer!
